vision/roadmap.md

```diff
@@ -105,6 +101,10 @@ Developers are able to build iOS apps in Swift using the [`xmtp-ios` SDK](/docs/
 
 Developers are able to build Flutter and mobile apps in Dart using the [`xmtp-flutter` SDK](/docs/client-sdk/dart/tutorials/quickstart).
 
+### Secure key storage module
+ 
```

Comment:
@saulmc - With v8, I believe the Keystore exists in app memory only. From what I understand from @neekolas, the next step is to allow it to exist in other more secure contexts, such as wallet apps.

Reply:
It lives in the app, but not just in memory. We make use of the app's LocalStorage which is persistent.